entrypoint.sh

```diff
@@ -39,7 +39,18 @@ fi
 # Run octodns-sync.
 echo "INFO: _config_path: ${_config_path}"
 if [ "${_doit}" = "--doit" ]; then
-  octodns-sync --config-file="${_config_path}" --doit
+  RESULT=$(octodns-sync --config-file="${_config_path}" --log-stream-stdout --doit)
 else
-  octodns-sync --config-file="${_config_path}"
+  RESULT=$(octodns-sync --config-file="${_config_path}" --log-stream-stdout)
 fi
+echo "$RESULT"
+
+# We want the result after a line starting with *
+SHORT_RESULT=$(echo "$RESULT" | sed -ne "/\*/,$ p")
+
+# We need to escape newlines or we only get first line of result
+SHORT_RESULT="${SHORT_RESULT//'%'/'%25'}"
+SHORT_RESULT="${SHORT_RESULT//$'\n'/'%0A'}"
+SHORT_RESULT="${SHORT_RESULT//$'\r'/'%0D'}"
+
+echo "::set-output name=result::$SHORT_RESULT"
```

Comment:
Thank you for sharing this change @thattommyhall !

I [think](https://github.com/github/octodns/blob/5c26f388519433a07e50d80abd02fef1a91e312f/octodns/cmds/args.py#L31-L33) adding `--log-stream-stdout` to the `octodns-sync` command will send output to stdout rather than to stderr. So adding the argument should not modify the behavior or output of the Action (which I think is good .. I've only been reviewing output in the Actions log itself, manually).

I'll have to learn more to understand what `echo "::set-output name=result::$SHORT_RESULT"` does. Guessing from your workflow definition it'll make the output directly available to subsequent steps in the workflow.

Yeah, let me study this a bit and make sure I understand what it'll do.

Reply:
There is a slight change in that the work of octodns is printed after rather than streamed to the console. I thought it worth it to pass the result along